src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -203,6 +202,114 @@ describe("Editor Page Component", () => {
         expect(saveProjectSpy).toBeCalledWith(expect.objectContaining(partialProject));
     });
 
+    it("When an image is updated the asset metadata is updated", async () => {
+        const testProject = MockFactory.createTestProject("TestProject");
+        const store = createStore(testProject, true);
+        const props = MockFactory.editorPageProps(testProject.id);
+        const wrapper = createComponent(store, props);
+        const imageAsset = testAssets[0];
+
+        await MockFactory.flushUi();
+        wrapper.update();
+
+        const editedImageAsset: IAssetMetadata = {
+            asset: imageAsset,
+            regions: [MockFactory.createMockRegion()],
+        };
+
+        const saveMock = assetServiceMock.prototype.save as jest.Mock;
+        saveMock.mockClear();
+
+        wrapper.find(Canvas).props().onAssetMetadataChanged(editedImageAsset);
+        await MockFactory.flushUi();
+
+        const editorPage = wrapper.find(EditorPage).childAt(0) as ReactWrapper<IEditorPageProps, IEditorPageState>;
```

Comment:
what's the difference between your version and doing

```
const editorPage: ReactWrapper<IEditorPageProps, IEditorPageState> = wrapper.find(EditorPage).childAt(0)
```

Reply:
Compiler doesn't the type of `.childAt(0)` so you have to cast it... i believe the way you mentioned would cause a TS compile error.  Will verify